docs/configuration_options.md

```diff
@@ -3489,6 +3489,15 @@ Forces GossipingPropertyFileSnitch (regardless `endpoint_snitch`) to simulate ra
 **type:** int
 
 
+## **rf_rack_valid_keyspaces** / SCT_RF_RACK_VALID_KEYSPACES
```

Comment:
Why do we need to introduce SCT configuration for a property in scylla yaml ?

Won't just changing the default scylla_yaml_append suffice?

I don't like the direction of adding so much for a single scylla configuration

Reply:
Right, what will hapeen with entry like this, will it be merged or overriden? 

```
 hinted_handoff: 'enabled'
 parallel_node_operations: true

append_scylla_yaml:
  yet_another_option: false
```

Reply:
Checked now, the `yet_another_option` gets appended to yaml along with `rf_rack_valid_keyspaces: true` when we use

```
 hinted_handoff: 'enabled'
 parallel_node_operations: true

append_scylla_yaml:
  yet_another_option: false
```